### PR TITLE
NOMRG test 0.23.1 build

### PR DESCRIPTION
**Pull request by adrinjalali**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% set version = "0.23.0" %}
+{% set version = "0.23.X" %}
```
Comment (jeremiedbb) [inline recipe/meta.yaml:1]: shouldn't it be 0.23.1 ?

Comment (adrinjalali) [reply recipe/meta.yaml:1]: No, I haven't tagged on the branch yet. I wanted to test if these builds succeed before tagging.

Comment (adrinjalali) [reply recipe/meta.yaml:1]: I wouldn't merge this PR anyway, I'd wait for the bot to create the PR once I draft a release on our repo.

Comment (jeremiedbb) [reply recipe/meta.yaml:1]: Ah ok, thanks. 

> I wanted to test if these builds succeed before tagging.

wise :)

**Pull request closed without merging**